docs/source/webui.rst

```diff
@@ -0,0 +1,118 @@
+Web UI
+======
+
+st2web is an Angular-based web application. It allows you to control the whole process of execution, from running an action to seeing the results of the execution. It also helps you to explore workflow executions up to the results of individual tasks. All in real time.
+
+Express installation
+--------------------
+
+Production version of st2web is pre-installed on st2express and any other st2_deploy installation. You can access the UI by accessing **/webui** endpoint of st2api server. For vagrant deployment of st2express, it would be http://172.168.90.50:9101/webui.
+
+It can also be installed by extracting the latest tar-ball from https://ops.stackstorm.net/releases/st2/0.8dev/webui/ into `/opt/stackstorm/static/webui`.
```

Comment:
Maybe we could say `https://ops.stackstorm.net/releases/st2/<version>/webui/ (e.g. https://ops.stackstorm.net/releases/st2/0.8dev/webui/)`, but not a biggie.